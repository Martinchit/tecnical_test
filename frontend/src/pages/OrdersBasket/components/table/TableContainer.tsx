import styled from 'styled-components';
import { Table } from 'react-bootstrap';

export const TableContainer = styled(Table)`
  padding-bottom: 100px;
`;
