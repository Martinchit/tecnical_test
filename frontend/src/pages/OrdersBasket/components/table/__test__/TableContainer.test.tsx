import React from 'react';
import { shallow } from 'enzyme';

import { TableContainer } from '../TableContainer';

describe('TableContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TableContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
