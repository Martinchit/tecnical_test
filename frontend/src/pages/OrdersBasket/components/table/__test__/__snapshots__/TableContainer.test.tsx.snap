// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableContainer renders correctly 1`] = `
<ForwardRef
  className="sc-AxjAm igBWVW"
/>
`;
